### DISCUSSION TITLE
The web interface gets disconnected early

### DISCUSSION THREAD

**Discussion by Exclusif66** [TheLastBen/fast-stable-diffusion#2446] [General]
For some reason the webui gets disconnected after a few minutes, which is annoying since you need to restart the whole boot up process again. This is a new problem that never happened before. I'm using Colab Pro.

**Comment by TheLastBen** [comment]
make sure you have compute units

**Comment by Exclusif66** [comment]
Thanks for your reply. I have compute units.



On Mon, 7 Aug 2023 at 17:22, Ben ***@***.***> wrote:

> make sure you have compute units
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/TheLastBen/fast-stable-diffusion/discussions/2446#discussioncomment-6660026>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZID5DVI2YOG6IDUX2QNTXTXUEB3XANCNFSM6AAAAAA3G77VM4>
> .
> You are receiving this because you authored the thread.Message ID:
> <TheLastBen/fast-stable-diffusion/repo-discussions/2446/comments/6660026@
> github.com>
>
-- 
Isak Mathiasson

**Comment by TheLastBen** [comment]
I'm not sure why colab would disconnect a pro account, maybe just a temporary bug

**Comment by kiazoo** [comment]
i got this error also. when i try to run last step its show transportation error output.
Maybe in your Gdrive has too many file in your drive. i try to delete file its work for me.

**Reply by Exclusif66** [reply]
The problem got resolved for me by using the More memory option for the session. I never had to use that option before this specific update.
